covid-19-support/src/App.vue

```diff
@@ -162,14 +165,19 @@ export default {
         }
       })
 
-      const dayFilter = dayFilters[this.getDay(this.day)]
+      var today = new Date().getDay()
+      var selectedDay = today
+      if (!this.isAnyDaySelected(this.day)) {
+        selectedDay = this.day
+      }
 
-      var open = markers.filter((c) => c[dayFilter].$t !== '0')
-      var closed = markers.filter((c) => c[dayFilter].$t == '0')
+      const dayFilter = dayFilters[this.getDay(selectedDay)]
+      const open = markers.filter((c) => c[dayFilter].$t !== '0')
+      const closed = markers.filter((c) => c[dayFilter].$t == '0')
 
       var retList = extend(
-        open.map((marker) => ({ marker, oc: true })),
-        closed.map((marker) => ({ marker, oc: false }))
+        open.map((marker) => ({ marker, oc: null })),
```

Comment:
This bit doesn't look right, the original code is just adding an additional parameter called OC ( open / close ) and either setting it as TRUE or FALSE. At this point everything in the OPEN object is open and everything in the CLOSED object is closed.

We then merge the two objects together and sort the items, this parameter then allows us to check if the item is open or closed :)